Mathlib/CategoryTheory/Monoidal/CommMon_.lean

```diff
@@ -133,33 +132,84 @@ instance : HasInitial (CommMon_ C) :=
 
 end CommMon_
 
-namespace CategoryTheory.Functor
+namespace CategoryTheory
+variable {C}
+  {D : Type u₂} [Category.{v₂} D] [MonoidalCategory D] [BraidedCategory D]
+  {E : Type u₃} [Category.{v₃} E] [MonoidalCategory E] [BraidedCategory E]
+  {F F' : C ⥤ D} [F.LaxBraided] [F'.LaxBraided] {G : D ⥤ E} [G.LaxBraided]
 
-variable {C} {D : Type u₂} [Category.{v₂} D] [MonoidalCategory.{v₂} D] [BraidedCategory.{v₂} D]
+namespace Functor
 
+variable (F) in
 /-- A lax braided functor takes commutative monoid objects to commutative monoid objects.
 
 That is, a lax braided functor `F : C ⥤ D` induces a functor `CommMon_ C ⥤ CommMon_ D`.
 -/
 @[simps!]
-def mapCommMon (F : C ⥤ D) [F.LaxBraided] : CommMon_ C ⥤ CommMon_ D where
+def mapCommMon : CommMon_ C ⥤ CommMon_ D where
   obj A :=
     { F.mapMon.obj A.toMon_ with
       mul_comm := by
         dsimp
         rw [← Functor.LaxBraided.braided_assoc, ← Functor.map_comp, A.mul_comm] }
   map f := F.mapMon.map f
 
-variable (C) (D)
+/-- The identity functor is also the identity on commutative monoid objects. -/
+@[simps!]
+noncomputable def mapCommMonIdIso : mapCommMon (𝟭 C) ≅ 𝟭 (CommMon_ C) :=
+  NatIso.ofComponents fun X ↦ CommMon_.mkIso (.refl _)
+
+/-- The composition functor is also the composition on commutative monoid objects. -/
+@[simps!]
+noncomputable def mapCommMonCompIso : (F ⋙ G).mapCommMon ≅ F.mapCommMon ⋙ G.mapCommMon :=
+  NatIso.ofComponents fun X ↦ CommMon_.mkIso (.refl _)
 
+variable (C D) in
 /-- `mapCommMon` is functorial in the lax braided functor. -/
 @[simps]
 def mapCommMonFunctor : LaxBraidedFunctor C D ⥤ CommMon_ C ⥤ CommMon_ D where
   obj F := F.mapCommMon
   map α := { app := fun A => { hom := α.hom.app A.X } }
   map_comp _ _ := rfl
 
-end CategoryTheory.Functor
+/-- Natural transformations between functors lift to monoid objects. -/
+@[simps!]
+noncomputable def mapCommMonNatTrans (f : F ⟶ F') [NatTrans.IsMonoidal f] :
+    F.mapCommMon ⟶ F'.mapCommMon where
+  app X := .mk (f.app _)
+
+/-- Natural isomorphisms between functors lift to monoid objects. -/
+@[simps!]
+noncomputable def mapCommMonNatIso (e : F ≅ F') [NatTrans.IsMonoidal e.hom] :
+    F.mapCommMon ≅ F'.mapCommMon :=
+  NatIso.ofComponents fun X ↦ CommMon_.mkIso (e.app _)
+
+end Functor
+
+open Functor
+
+namespace Adjunction
+variable {F : C ⥤ D} {G : D ⥤ C} (a : F ⊣ G) [F.Braided] [G.LaxBraided] [a.IsMonoidal]
+
+/-- An adjunction of braided functors lifts to an adjunction of their lifts to commutative monoid
+objects. -/
+@[simps!] noncomputable def mapCommMon : F.mapCommMon ⊣ G.mapCommMon where
+  unit := mapCommMonIdIso.inv ≫ mapCommMonNatTrans a.unit ≫ mapCommMonCompIso.hom
+  counit := mapCommMonCompIso.inv ≫ mapCommMonNatTrans a.counit ≫ mapCommMonIdIso.hom
+
+end Adjunction
+
+namespace Equivalence
+
+/-- An equivalence of categories lifts to an equivalence of their commutative monoid objects. -/
+noncomputable def mapCommMon (e : C ≌ D) [e.functor.Braided] [e.inverse.Braided] [e.IsMonoidal] :
```

Comment:
```suggestion
@[simps!]
noncomputable def mapCommMon (e : C ≌ D) [e.functor.Braided] [e.inverse.Braided] [e.IsMonoidal] :
```
It's usually not a good idea to have definitions in the library without lemmas describing it.